packages/components/pie-list/src/index.ts

```diff
@@ -12,11 +18,50 @@ const componentSelector = 'pie-list';
 
 /**
  * @tagname pie-list
+ *
+ * PIE List component for displaying collections of related items.
```

Comment:
we will want to document the slot(s) in this comment too - you can see examples in other components for the correct formatting